services/github/github-actions-workflow-status.service.js

```diff
@@ -1,46 +1,22 @@
 import Joi from 'joi'
 import { isBuildStatus, renderBuildStatusBadge } from '../build-status.js'
-import { NotFound } from '../index.js'
-import { GithubAuthV3Service } from './github-auth-service.js'
-import { documentation, errorMessagesFor } from './github-helpers.js'
+import { BaseSvgScrapingService } from '../index.js'
+import { documentation } from './github-helpers.js'
 
 const schema = Joi.object({
-  workflow_runs: Joi.array()
-    .items(
-      Joi.object({
-        status: Joi.equal(
-          'completed',
-          'action_required',
-          'cancelled',
-          'failure',
-          'neutral',
-          'skipped',
-          'stale',
-          'success',
-          'timed_out',
-          'in_progress',
-          'queued',
-          'requested',
-          'waiting'
-        ).required(),
-        conclusion: Joi.alternatives()
-          .try(isBuildStatus, Joi.equal('no status'), null)
-          .required(),
-      })
-    )
-    .required()
-    .min(0)
-    .max(1),
+  message: Joi.alternatives()
+    .try(isBuildStatus, Joi.equal('no status'))
+    .required(),
 }).required()
 
 const queryParamSchema = Joi.object({
   event: Joi.string(),
-  branch: Joi.string().required(),
+  branch: Joi.string(),
```

Comment:
Two things here:
1. As long as it isn't `.required()`, this works for an entirely numeric branch name. Refs #8734 and #8720
2. This puts us back to a situation where `branch` is optional. If we think we might want to do this as a temporary solution, we could:
    - keep branch required
    - if the user passes `*`, we request `/badge.svg` without a `?branch=` param

    and then that way, our options are slightly more open. If we make `branch` optional again now, we're probably committed to this implementation forever.

Reply:
> * if the user passes `*`, we request `/badge.svg` without a `?branch=` param

In the event that the above solution _does_ end up being temporary, would it not be more prudent to request `/badge.svg` if the branch parameter is _empty_ rather than requiring a specific branch value? Unless I'm missing something.